openssl/src/cms.rs

```diff
@@ -219,14 +219,65 @@ impl CmsContentInfo {
             Ok(CmsContentInfo::from_ptr(cms))
         }
     }
+
+    /// Verify this CmsContentInfo's signature, given a stack of certificates
+    /// in certs, an X509 store in store. If the signature is detached, the
+    /// data can be passed in data. The data sans signature will be copied
+    /// into output_data if it is present.
+    ///
+    /// OpenSSL documentation at [`CMS_verify`]
+    ///
+    /// [`CMS_verify`]: https://www.openssl.org/docs/manmaster/man3/CMS_verify.html
+    pub fn verify(
+        &mut self,
+        certs: Option<&StackRef<X509>>,
```

Comment:
The man page for CMS_verify doesn't talk about certs and store being allowed to be null at all - maybe they should just be required?

Reply:
The man page (https://www.openssl.org/docs/manmaster/man3/CMS_verify.html) mentions a null certs parameter in this sentence:

_...An attempt is made to locate all the signing certificate(s), first looking in the certs parameter (if it is not NULL) ..._

 and there is the option CMS_NO_SIGNER_CERT_VERIFY which disables certificate verification altogether.

Therefore, I think the optional parameters make sense. Nevertheless, the distinction is just a formal one, because the caller can always create an empty vector or an empty store.  If you insist, I can change them to required, just let me know.

Reply:
Ah good find. Keeping `certs` nullable seems fine, but I'd lean towards requiring `store` since its nullability isn't mentioned in the docs and at least in 1.0.1 they assume it's present (I'd also hope people aren't using NO_SIGNER_CERT_VERIFY too often!).